src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1513,30 +1447,92 @@ inline RppStatus rpp_store4_f32pln3_to_u8pkd3(Rpp8u* dstPtr, __m128* p)
 
 inline RppStatus rpp_store4_f32pln3_to_u8pln3(Rpp8u* dstRPtr, Rpp8u* dstGPtr, Rpp8u* dstBPtr, __m128* p)
```

Comment:
This one is similar to rpp_store12_f32pln3_to_f32pln3() so lets reference with the number 12. 4 for each color.